### ISSUE TITLE
cannot find module providing package github.com/prometheus/common/log

### ISSUE THREAD

**Issue by mweinelt** [czerwonk/junos_exporter#145]
The title says it all. 

```
❯ go version
go version go1.16.6 linux/amd64
❯ GOPATH=$(pwd)/go go get -u github.com/czerwonk/junos_exporter
go: downloading github.com/czerwonk/junos_exporter v0.7.1
go: downloading github.com/prometheus/common v0.29.0
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/prometheus/client_golang v1.11.0
go: downloading golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
go: downloading github.com/prometheus/common v0.26.0
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/golang/protobuf v1.4.3
go: downloading github.com/prometheus/procfs v0.6.0
go: downloading golang.org/x/sys v0.0.0-20210615035016-665e8c7367d1
go: downloading github.com/sirupsen/logrus v1.6.0
go: downloading gopkg.in/alecthomas/kingpin.v2 v2.2.6
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/prometheus/procfs v0.7.1
go: downloading github.com/golang/protobuf v1.5.2
go: downloading github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
go: downloading github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
go: downloading google.golang.org/protobuf v1.26.0-rc.1
go: downloading github.com/sirupsen/logrus v1.8.1
go: downloading golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
go: downloading github.com/alecthomas/units v0.0.0-20210208195552-ff826a37aa15
go: downloading google.golang.org/protobuf v1.27.1
go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.3
github.com/czerwonk/junos_exporter imports
	github.com/prometheus/common/log: cannot find module providing package github.com/prometheus/common/log
```

**Comment by mweinelt** [comment]
The reason this happens is that the log component has been deprecated and removed in 0.27.0.

https://github.com/prometheus/common/pull/306

**Comment by czerwonk** [comment]
Hi, that's correct and that is also the reason i switched to logrus in e5af1e500b87ef32f21a8e52eb89de219ea91fd9. I will look out for other references and keep you updated.

**Comment by czerwonk** [comment]
Ok, if I run it on a fresh go install (in my case golang container) without using -u it works. With -u it does not. Can you please verify if intallation without using -u works for you too?

**Comment by mweinelt** [comment]
Yep, that works.

**Comment by czerwonk** [comment]
Fixed it  by using v0.26.0 insted of v0.29.0. The client packge v1.11.0 uses v0.26.0 itself. So it is not compatible
